src/unfold.js

```diff
@@ -15,23 +15,25 @@ var _curry2 = require('./internal/_curry2');
  * @category List
  * @sig (a -> [b]) -> * -> [b]
```

Comment:
It should look similar to the signature of [`S.unfoldr`][1], though in Ramda the type of `['foo', 'bar', 'baz']` is written `[String]` rather than `Array String`.


[1]: https://sanctuary.js.org/#unfoldr